ec-europa/eubfr-data-lake
# feat(serverless-plugin-elasticsearch-index): add plugin - EUBFR-101

Ready for review, there shouldn't be any breaking changes.

To test:
- run `yarn` to trigger lerna for linking new dependencies
- make deployment for a **new** stage
- verify that your stage-named index and its mapping is present at [dev-projects](https://eu-central-1.console.aws.amazon.com/es/home?region=eu-central-1#dev-projects:dashboard) domain
